extensions/amp-story-360/0.1/test/validator-amp-story-360.html

```diff
@@ -0,0 +1,50 @@
+<!--
+  Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Update to 2020

Reply:
Done

Reply:
I don't see the changes in my view. Did you push your changes to github?

Reply:
GitHub has been acting up all day, here's Tom's commit https://github.com/ampproject/amphtml/pull/29235/commits/7004226f77b6a90d529bfae6195fb3caa2a8905e
I assume it'll show up in the diff later

Reply:
Actually I missed one of the 2018's in that commit. Now fixed with 7ef2edb. Sorry about that.